jupyterlab_latex/__init__.py

```diff
@@ -199,7 +203,9 @@ def get(self, path = ''):
         Given a path, run LaTeX, cleanup, and respond when done.
         """
         # Get access to the notebook config object
-        tex_file_path = os.path.abspath(path.strip('/'))
+        tex_file_path = os.path.abspath(
+            os.path.join(self.notebook_dir, path.strip('/'))
```

Comment:
I *think* that `notebook_dir` is already an absolute path (according to my logging)

Reply:
kk i'll change that then!